docker/metadata/openmetadata-start.sh

```diff
@@ -10,10 +10,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-MYSQL="${MYSQL_HOST:-mysql}":"${MYSQL_PORT:-3306}"
-while ! nc -z -w 5 "${MYSQL}";
-  do echo "Trying to connect to ${MYSQL}"; sleep 5;
-done
+echo "Starting OpenMetadata Server";
```

Comment:
I will need to test out this change with OpenMetadata Helm Charts, creating a local docker image! Let me get back here testing post that!

Reply:
Feel free to put it back if OpenMetadata Helm Charts test fails after this change.

Reply:
Tested out this change! Looks good on Kubernetes as well! Let's roll in these changes! :)